fenix/views/android_privacy_metrics_cat.view.lkml

```diff
@@ -0,0 +1,75 @@
+
+view: android_privacy_metrics_cat {
```

Comment:
could this be combined with the numeric features? one way to do that is to split out each possible value of (for example) "preferences_etp_custom_cookies_selection" into a different measure - the count of clients who have that value. however, this might not be great if there are lots of possible values.

i think the main thing with the way you have it right now is it might make it harder for others to find the specific measure they are looking for.